cmd/main.go

```diff
@@ -72,9 +73,16 @@ func main() {
 			if len(args) != 3{
 				return errors.New("Wrong number of Input arguments expected 3 got "+strconv.Itoa(len(args)))
 			}
-			if (tlsConn == false && insecure == true) {
+			if (args[1][0:5] != "amqps" && insecure) {
```

Comment:
I don't think we need special support here, as this is already handled by the AMQP library:

https://github.com/vcabbage/amqp/blob/5a75e78e59d314879580d023e789feb1d4d1d22b/client.go#L77-L80